Sandbox/Enums/Enums.cs

```diff
@@ -0,0 +1,15 @@
+using System;
+
+namespace Enums
+{
+    public class Enums
+    {
+
+        public enum Day {Monday, Tuesday, Wednesday,Thursday, Friday, Saturday, Sunday };
+        public enum Month {January, February,March, April, MarshalByRefObject, June, July, August, September, October, November, December};
```

Comment:
If you are curious, try setting the value of _one_ of these to a high number, like 100, and see how that affects the numbering of the other values